### PR TITLE
Set Project language level to JDK6

### LINKED ISSUE #67: Remove JDK 7 requirement
We add JDK 7 requirement for an exception handling syntax available only in JDK 7.
As a fundamental library, it would be better to not have JDK 7 requirement. We don't have to use that exception handling syntax.

### PR DESCRIPTION
**Pull request by winningsix**
This fixes #67

### CODE REVIEW

**Comment by sundapeng** [comment]
Please also update the .travis.yml
```
language: java
jdk:
  - openjdk6
  - oraclejdk7
script:
  - mvn apache-rat:check
  - mvn clean test
  - jdk_switcher use oraclejdk7
  - mvn clean test -Pfindbugs
```

File: pom.xml [hunk 1]
```diff
@@ -94,6 +93,24 @@
       </properties>
     </profile>
     <profile>
+      <id>jdk6</id>
```
Comment (sundapeng) [inline pom.xml]: I think no matter what jdk version is, the property should be 1.6

**Comment by sundapeng** [comment]
Thank @winningsix , I left some comments.

File: pom.xml (hunk 1 above)
Comment (jerrychenhf) [inline pom.xml]: I tend to agree with Dapeng that we can use the least jdk version which is 1.6.
We don't need to add the different profiles for different JDK versions.

**Comment by jerrychenhf** [comment]
Thank @winningsix for this improvements.
LGTM after remove the added profiles for jdk.

**Pull request merged**